### ISSUE TITLE
Make unavailable units/lessons accessible

### ISSUE THREAD

**Issue by jack-skerrett-oak** [oaknational/Oak-Web-Application#1329]
a11y accessible, not accessible, you can't access them, they are unavailable.

Currently, to a screen reader user, they are not there unless you nav through the page line by line, which most users will not.

**Comment by jack-skerrett-oak** [comment]
I don't think this is an issue, it isn't a link.
